pkg/envoy/ads/stream.go

```diff
@@ -20,6 +22,42 @@ import (
 	"github.com/openservicemesh/osm/pkg/utils"
 )
 
+// This is the workqueue Job implementation to issue a Send Response to a proxy
+type sendJob struct {
+	// Send Response parametres
```

Comment:
```suggestion
	// Send Response parameters
```

I don't know that reading this comment adds any value. I don't think I learned much from it about what the struct does.

Reply:
removed